src/Zicht/Bundle/UrlBundle/Aliasing/Listener.php

```diff
@@ -35,6 +41,21 @@ public function __construct(Aliasing $aliasing, RouterListener $router)
     {
         $this->aliasing = $aliasing;
         $this->router = $router;
+        $this->annotationReader = new AnnotationReader();
+    }
+
+    public function onKernelController(Event\FilterControllerEvent $event)
```

Comment:
Als je deze functionaliteit wilt toevoegen, vind ik dat je hiervoor een aparte listener service moet aanmaken. Je gaat nu header annotations uitlezen in een listener die aliases toepast. Dat vind ik niet logisch